conf/modules/modules.config

```diff
@@ -87,6 +87,24 @@ process {
         }
     }
 
+    // CONCATENATED, SORT, UNANNOTATED VCFS
+    withName: 'GERMLINE_VCFS_CONCAT_SORT'{
+        ext.prefix       = { "${meta.id}.germline" }
+        publishDir       = [
+            mode: params.publish_dir_mode,
+                path: { "${params.outdir}/variant_calling/concat/${meta.id}/" }
+        ]
+    }
+
+    withName: 'TABIX_GERMLINE_VCFS_CONCAT_SORT'{
+        ext.prefix       = { "${meta.id}.germline" }
+        publishDir       = [
+            mode: params.publish_dir_mode,
+                path: { "${params.outdir}/variant_calling/concat/${meta.id}/" }
+        ]
+    }
+
```

Comment:
maybe this can be moved into a separate config

Reply:
Sure, I'll see if I can figure out where to put those things 😊

Reply:
How about I put that configuration into a new config-file under `conf/` called something like:

`variant_calling.config`,
`post_variant_calling.config`,
`vcf_postprocessing.config`, or
`postprocessing_vcfs.config`

Reply:
yes, you vcfs are not annotated, right?

Reply:
right, the vcfs are not annotated.

Reply:
okay, I'll name it `post_variant_calling`.